<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>235</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>기동5중대 근무배치 프로그램(Ver.2.0)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../res/myres.qrc">
    <normaloff>:/icon/경기지방남부경찰청 아이콘.ico</normaloff>:/icon/경기지방남부경찰청 아이콘.ico</iconset>
  </property>
  <widget class="QTabWidget" name="tab_main">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>381</width>
     <height>141</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="tabBarAutoHide">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="tab_new">
    <attribute name="title">
     <string>근무표 생성</string>
    </attribute>
    <widget class="QLineEdit" name="li_worker">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>첫 근무자 입력</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_new">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_del">
    <attribute name="title">
     <string>'/' 제거</string>
    </attribute>
    <widget class="QLineEdit" name="li_time">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>날짜-시간 입력</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bt_time">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_2">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>80</y>
       <width>118</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>근무표 유형</string>
   </property>
   <widget class="QRadioButton" name="bt_holiday">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>공휴일</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bt_weekend">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>주말</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="bt_weekday">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>평일</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>tab_main</zorder>
 </widget>
 <resources>
  <include location="../res/myres.qrc"/>
 </resources>
 <connections/>
</ui>
